lahja/endpoint.py

```diff
@@ -100,8 +144,11 @@ def connect_no_wait(self, loop: Optional[asyncio.AbstractEventLoop] = None) -> N
 
         self._running = True
 
-    async def connect(self, loop: Optional[asyncio.AbstractEventLoop] = None) -> None:
-        self.connect_no_wait(loop)
+    async def connect(self,
```

Comment:
The `connect_*` methods and the `connect_to_endpoints_*` methods aren't very visually distinct, it might be easy to accidentally mistake one for the other. Maybe they could be renamed to something further apart?

- `start_serving` and `connect_to_endpoints`
- `serve_local_endpoint` and `connect_remote_endpoints`

Reply:
>* `start_serving` and `connect_to_endpoints`

Let's do this!